apps/sage/account/NanoList.tsx

```diff
@@ -18,6 +18,7 @@ export default function NanoList() {
     REGAPI.register()
       .then((data) => {
         setRegKey(true)
+        setErrMsg('')
```

Comment:
Better UX would be to remove the error message whenever there is form change, but I'll go ahead and merged this.  There should still be some sort of state for "in progress" whenever an ajax call is made.  Finally, the button rendering could be 'simplified' to update the `disabled` attribute and `innerHTML` instead of rendering the button element, but this works.  I could these later.